apps/src/lab2/views/components/instructions.module.scss

```diff
@@ -133,15 +133,25 @@
 
 .textContent {
   height: 100%;
-  padding: 15px 20px 5px 20px;
+  padding: 15px 10px 5px;
   box-sizing: border-box;
   display: flex;
   flex-direction: column;
 }
 
-.scrollArea {
+%scrolling-content {
```

Comment:
nit: Should this be camel case?

Reply:
I actually updated it to kebab-case bc of the linter. Apparently, the convention for css variables is to use kebab-case.

Reply:
oh gotcha - that makes sense. I've seen other css variables that look that way; this just looked a little funny to me at a glance.